fuse_core/src/uuid.cpp

```diff
@@ -86,34 +86,16 @@ namespace uuid
 // DEALINGS IN THE SOFTWARE.
 UUID generate()
 {
-  static std::random_device rd;
-  static std::mt19937 generator(rd());
-  static std::uniform_int_distribution<uint64_t> distibution;
-  static std::mutex distribution_mutex;
+  static boost::uuids::random_generator generator;
```

Comment:
I think you have to update the comment above: https://github.com/locusrobotics/fuse/pull/171/files#diff-8781d87c9486b0c374ec2a4c6adec410R54